functions/processQueueItem.js

```diff
@@ -26,9 +38,42 @@ module.exports.handler = async function(
     positive_email,
     notes,
     image
-  } = JSON.parse(Body);
+  } = JSON.parse(body);
+
+  console.log("Processing queue item", id);
 
   try {
+    // Pause if we're at the rate limit for current expiration window
+    let rateLimited = false;
+    do {
```

Comment:
The way this rate limiting works is that the function pauses briefly if a sliding window query from the hitrate table yields too many hits to proceed.

It can repeat this scan-and-pause loop for up to 60 seconds per current configuration. There's also a concurrency limit of 5, so that should mean only up to 5 items will hang around in a rate limit pause loop until there's free capacity to process items.